extensions/omni_txn/docs/variables.md

```diff
@@ -1,5 +1,10 @@
 # Transaction Variables
 
+!!! warning "This functionality has been moved"
```

Comment:
I am happy with even removing `omni_txn` completely in this PR. Not sure we need to worry about backward compatibility at this point.

Reply:
We have users using it right now, don't want to break it right away